src/main/resources/org/jenkins/plugins/lockableresources/LockStep/help-maxWaiting.html

```diff
@@ -0,0 +1,11 @@
+<div>
+	<p>
+		By default builds a queued without any restriction on waiting to lock a common resource
+		Setting this parameter a limit is forced in the number of waiting builds. Any build trying to lock after
+		the limit is reached will trigger a cancel round trip which will abort the older waiting build.
+	</p>
+	<p>
+		For example, if builds #3 and #4 are waiting to lock the resource and build #5 reaches the lock step,
+		then build #3 will be cancelled in favor of #5 which is newer.
```

Comment:
Do we actually need this feature? To my mind the `milestone` step is supposed to be taking care of that kind of ordering. In other words, 3, 4, and 5 would all wait for the lock. You _can_ take build number into consideration when deciding who gets the lock when it is released, giving preference to 5; then when 5 reaches its next milestone, 3 and 4 would get canceled automatically.